File: Services/TokenService/Models/UserModel.cs
using Microsoft.AspNetCore.Mvc.RazorPages;

namespace TokenService.Models
{
    public class UserModel : BaseModel
    { 
        public string Email { get; set; }
        public int CredentialsId { get; set; }
        public int RoleId { get; set; }
        public string RoleName { get; set; }
        public DateTime RegistrationTime { get; set; }
    }
}

File: Client/src/app/shared/models/user.model.ts
import { EI } from "./EI.model";

export class User {
    id: number;
    email: string;
    registrationTime?: Date;
    credentialsId?: number;
    roleId?: number;
    roleName?: string;
    eis?: EI[];
    constructor(id:number, email: string, nickname: string){
        this.id = id;
        this.email = email
    }
}